### ISSUE TITLE
Experimental starts games in a frozen state

### ISSUE THREAD

**Issue by yjfguhd** [ValveSoftware/Proton#7009]
The latest Experimental has the following effect on three games that I've tested, maybe all my collection. Proton 8.0-3 is fine.

The game seems to initialise OK when I press Run. However, the very first thing that the game shows will be frozen in time, sometimes it will be frozen in a faded state. For example, if the game normally fades in a logo, the first thing I will see is that logo, frozen, at some stage during fade-in.

For example, Ori and the Blind Forest: Definitive Edition usually first shows the Ori character running on the spot as the game loads. With Experimental, I get this static (only darker than shown here, because I could only get this screenshot by using Proton 8.0-3):

![20230812142121_1](https://github.com/ValveSoftware/Proton/assets/44527569/4581462d-2e3d-438c-b028-68bfd84fcbc9)

The same goes for INSIDE (darker again, this screenshot was taken using Proton 8.0-3 when the game works fine):

![20230812142839_1](https://github.com/ValveSoftware/Proton/assets/44527569/f3906651-aca0-4b32-a4fe-9c1ae79669cd)

And with Euro Truck Simulator 2, which apparently doesn't fade in, but rather immediately shows a background image before loading the main menu. This time the background image appears, frozen. The mouse cursor is unresponsive:

![20230812140207_1](https://github.com/ValveSoftware/Proton/assets/44527569/91ad94e5-e14d-4028-b5c6-2a9a17b59439)

In all cases I have waited a few minutes. Sometimes I'll catch the screen moving a bit, as if showing the next frame, but again, frozen. Shift-tab and F12 (for a screenshot) are unresponsive. I am able to alt-tab to other applications, and I can kill the game with alt-F4. I'm using Openbox in XFCE on Void Linux. I think this started with the Experimental version before the 10th August update.

I attach PROTON_LOG files for both working (8.0-3) and Experimental states.

[steam-227300-8.0-3.log](https://github.com/ValveSoftware/Proton/files/12327505/steam-227300-8.0-3.log)
[steam-227300-experimental.log](https://github.com/ValveSoftware/Proton/files/12327506/steam-227300-experimental.log)
[steam-304430-8.0-3.log](https://github.com/ValveSoftware/Proton/files/12327507/steam-304430-8.0-3.log)
[steam-304430-experimental.log](https://github.com/ValveSoftware/Proton/files/12327508/steam-304430-experimental.log)
[steam-387290-8.0-3.log](https://github.com/ValveSoftware/Proton/files/12327509/steam-387290-8.0-3.log)
[steam-387290-experimental.log](https://github.com/ValveSoftware/Proton/files/12327510/steam-387290-experimental.log)

**Comment by Xenonobi** [comment]
> And with Euro Truck Simulator 2, which apparently doesn't fade in

Euro Truck Simulator 2 is native available for SteamOS + Linux as well as American Truck Simulator.

I have both Games in my Collection and i can assure you they are working out of the Box, and
there was never a need for me to activate Proton to play both Games. :)

**Comment by yjfguhd** [comment]
> I have both Games in my Collection and i can assure you they are working out of the Box, and there was never a need for me to activate Proton to play both Games. :)

Glad to hear it. I get much better performance using the Proton version than native version of these games, but this is aside from the issue.

**Comment by klaritan** [comment]
I'm getting this issue with Shotgun Farmers as well. Just freezes on the loading screen. It's a Unity game.

**Comment by yjfguhd** [comment]
Well, this is strange. It is no longer happening for me, and I'm still using the same Proton 1691694710 experimental-8.0-20230810b.

**Comment by yjfguhd** [comment]
This is happening again. Proton: 1692722216 experimental-8.0-20230822, and also GE-Proton8-8. Proton 8.0-3 is still OK.

**Comment by vktr2b** [comment]
Now that Proton 8.0-3 has been updated to Proton 8.0-4, I have started to experience the same issue. Is there any update to this issue or a workaround?

```
(wine:1065): GStreamer-WARNING **: 22:12:21.668: Failed to load plugin '/mnt/Seagate/SteamLibrary/steamapps/common/Proton 8.0/dist/lib64/gstreamer-1.0/libgstisomp4.so': /mnt/Seagate/SteamLibrary/steamapps/common/Proton 8.0/dist/lib64/gstreamer-1.0/libgstisomp4.so: wrong ELF class: ELFCLASS64

(wine:1065): GStreamer-WARNING **: 22:12:21.668: Failed to load plugin '/mnt/Seagate/SteamLibrary/steamapps/common/Proton 8.0/dist/lib64/gstreamer-1.0/libgstwavparse.so': /mnt/Seagate/SteamLibrary/steamapps/common/Proton 8.0/dist/lib64/gstreamer-1.0/libgstwavparse.so: wrong ELF class: ELFCLASS64
```

[steam-1868140.log](https://github.com/ValveSoftware/Proton/files/12841770/steam-1868140.log)

**Comment by vktr2b** [comment]
++ Adding some logs from the console that might be interesting, when running steam through a command line 


```
/bin/sh\0-c\0PROTON_LOG=1 /home/user/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_32/reaper SteamLaunch AppId=774361 -- /home/user/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_32/steam-launch-wrapper -- '/home/user/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/SteamLinuxRuntime_sniper'/_v2-entry-point --verb=waitforexitandrun -- '/mnt/Seagate/SteamLibrary/steamapps/common/Proton 8.0'/proton waitforexitandrun  '/mnt/Seagate/SteamLibrary/steamapps/common/Blasphemous/Blasphemous.exe'\0
chdir "/mnt/Seagate/SteamLibrary/steamapps/common/Blasphemous"
ERROR: ld.so: object '/home/user/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/home/user/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_64/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS64): ignored.
ERROR: ld.so: object '/home/user/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/home/user/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/home/user/.var/app/com.valvesoftware.Steam/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
WARNING: discarding _NET_WM_PID 1170 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 1170 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 1864 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 1990 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 13390 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 24778 as invalid for X11 window - use specialized XCB_X11_TO_PID function!
WARNING: discarding _NET_WM_PID 43050 as invalid for X11 window - use specialized XCB_X11_TO_PID function!

```

**Comment by yjfguhd** [comment]
Aren't those the usual errors you get as Steam tries to load 32-bit libraries before 64-bit (or vice versa)?

**Comment by vktr2b** [comment]
Yeah you are right, my bad those are usual errors, I've tested with Proton 7.0 and those were there as well. I've tested so many things I was just glad I found something that might lead to a resolution but should have verified a bit more.

At the moment I think I'm gonna stick with proton 7.0 and check back to 8.X in a while to see if it is fixed.

**Comment by DarielVR** [comment]
Are you using Nvidia? If that's the case look up if you have [ForceCompositionPipeline turned ON on the nvidia settings.](https://www.reddit.com/r/linux_gaming/comments/151mb2y/comment/jux7z1k/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button)

**Comment by yjfguhd** [comment]
I do indeed, and I'll give that a try if it happens again—thanks! The odd thing is that it can one day happen, the next not, even if Steam and Proton haven't updated.

**Comment by yjfguhd** [comment]
It does seem to work around it for me, but of course it means I'm subject to screen tearing.
